docs/2.deploy/20.providers/cloudflare.md

```diff
@@ -264,12 +264,39 @@ defineEventHandler(async (event) => {
 })
 ```
 
-### Access to the bindings in local dev
+## Dev Preset
 
-> [!NOTE]
-> The `nitro-cloudflare-dev` module is experimental. The Nitro team is looking into a more native integration  which could in the near future make the module unneeded.
+Cloudflare preset can be enabled in development mode for production environment emulation and accessing to the bindings in local dev
+
+In order to enable dev preset, make sure using latest nitro version (>=2.12) and install [`wrangler`](https://npmjs.com/package/wrangler) as a dependency.
+
+:pm-install{name="-D wrangler"}
 
-In order to access bindings in dev mode we start by defining the bindings. You can do this in a `wrangler.toml`/`wrangler.json` file, or directly in your Nitro config under `cloudflare.wrangler` (accepts the same type as `wrangler.json`).
+Then, update config:
+
+```ts [nitro.config.ts]
+export default defineNitroConfig({
+    compatibilityDate: "2024-07-13", // or "latest"
```

Comment:
Do you want to use 2024-09-19 here and below?

ref `:important To use Workers with Static Assets, you need a Nitro compatibility date set to 2024-09-19 or later. ::` earlier in this doc